pkg/chains/formats/slsa/internal/material/material.go

```diff
@@ -143,18 +163,12 @@ func Materials(ctx context.Context, tro *objects.TaskRunObject) ([]common.Proven
 			mats = append(mats, m)
 		}
 	}
-
-	// remove duplicate materials
-	mats, err := RemoveDuplicateMaterials(mats)
-	if err != nil {
-		return mats, err
-	}
-	return mats, nil
+	return mats
 }
 
-// gitInfo scans over the input parameters and looks for parameters
+// GitInfo scans over the input parameters and looks for parameters
```

Comment:
This should have been done when this function was added. But can we please refine this docstring a bit? It just says "look for parameters". Without looking into the code, it's hard to know what params were inspected by this function. Thank you!!!